### PR TITLE
feat: add friendly name to spec

### PR DESCRIPTION
**Pull request by J0**
## What kind of change does this PR introduce?

- Corresponding docs update for: https://github.com/supabase/gotrue/pull/1277

### CODE REVIEW

File: apps/www/pages/contact/mfa.tsx
Comment (hf) [inline apps/www/pages/contact/mfa.tsx:1]: File is deleted?

Comment (J0) [reply apps/www/pages/contact/mfa.tsx:1]: Yup that is intended I think we no longer need the early access page as it's available to all. Will wait for website team to confirm if it's ok to delete though

**Comment by J0** [comment]
Actually, the Auth deploy needs to go out before this can be merged. Reverting to draft

File: apps/www/pages/contact/mfa.tsx
Comment (J0) [reply apps/www/pages/contact/mfa.tsx:1]: Decided to do in another PR as it's not relevant to this one

**Review by charislam — APPROVED** [review]

**Pull request merged**